Vocable/Features/Settings/EditCategories/EditCategoryDetailViewController.swift

```diff
@@ -9,118 +9,209 @@
-        collectionView.register(UINib(nibName: "EditCategoryToggleCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: EditCategoryToggleCollectionViewCell.reuseIdentifier)
-        collectionView.register(UINib(nibName: "EditCategoryRemoveCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: EditCategoryRemoveCollectionViewCell.reuseIdentifier)
-        collectionView.register(UINib(nibName: "SettingsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: SettingsCollectionViewCell.reuseIdentifier)
         collectionView.backgroundColor = .collectionViewBackgroundColor
+        collectionView.delaysContentTouches = false
+
+        collectionView.collectionViewLayout = UICollectionViewCompositionalLayout { [weak self] (sectionIndex, environment) -> NSCollectionLayoutSection? in
 
-        collectionView.collectionViewLayout = UICollectionViewCompositionalLayout { (sectionIndex, environment) -> NSCollectionLayoutSection? in
+            let section = self?.dataSource?.snapshot().sectionIdentifiers[sectionIndex]
 
-            let section = self.dataSource.snapshot().sectionIdentifiers[sectionIndex]
             switch section {
-            case .header:
-                return self.headerSection(environment: environment)
-            case .body:
-                return self.bodySection(environment: environment)
+            case .editCategory:
+                return self?.defaultSection(environment: environment)
+            case .editPhrase, .removeCategory:
+                return self?.footerSection(environment: environment)
+            case .none:
+                return nil
             }
         }
     }
 
-    private func headerSection(environment: NSCollectionLayoutEnvironment) -> NSCollectionLayoutSection {
-
-        let itemHeightDimension = NSCollectionLayoutDimension.absolute(50)
-        let itemWidthDimension = NSCollectionLayoutDimension.fractionalWidth(1.0)
-
-        let itemSize = NSCollectionLayoutSize(widthDimension: itemWidthDimension, heightDimension: itemHeightDimension)
-        let item = NSCollectionLayoutItem(layoutSize: itemSize)
-        let groupSize = NSCollectionLayoutSize(widthDimension: itemWidthDimension, heightDimension: itemHeightDimension)
-        let group = NSCollectionLayoutGroup.horizontal(layoutSize: groupSize, subitem: item, count: 1)
-        group.interItemSpacing = .fixed(8)
-
-        let section = NSCollectionLayoutSection(group: group)
-        section.interGroupSpacing = 8
-        section.contentInsets = sectionInsets(for: environment)
-        section.contentInsets.top = 8
+    private func footerSection(environment: NSCollectionLayoutEnvironment) -> NSCollectionLayoutSection {
+        let section = defaultSection(environment: environment)
+        section.contentInsets.top += sizeClass == .hCompact_vRegular ? 32 : 16
```

Comment:
```suggestion
        section.contentInsets.top += (sizeClass == .hCompact_vRegular) ? 32 : 16
```
My eyes were struggling with order of operations for a second here 😄